Documentation/En/Index.xyl

```diff
@@ -640,7 +641,7 @@ public function case_sum()
   to a <strong>virtual file</strong> or a <strong>directory</strong>.
   Additional <strong>query strings</strong> can be present to specify more
   information, such as permissions, access time etc. The parent-child relation
-  is automatically created, not need to specify that a file is a child of a
+  is automatically created, it is not necessary to specify that a file is a child of a
```

Comment:
in that case `there is no need to specify ...` but `not necessary` hold the same meaning.